coderadar-plugin-api/src/main/java/io/reflectoring/coderadar/plugin/api/Finding.java

```diff
@@ -16,13 +16,16 @@
 
     private Integer charEnd = null;
 
+    private String message = null;
+
     /**
      * Marks a finding over multiple complete lines.
      *
      * @param lineStart the line number where the finding starts.
      * @param lineEnd   the line number where the finding ends.
+     * @param message the message of the metric.
      */
-    public Finding(Integer lineStart, Integer lineEnd) {
+    public Finding(Integer lineStart, Integer lineEnd, String message) {
```

Comment:
The parameter `message` is not used here. Also, the constructor is never used.

Reply:
Done